packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -285,7 +287,12 @@ export default function PostOptionsMenu({
       label: 'Hide',
       action: onHidePost,
     },
-    {
+  ];
+
+  const bookmarkOnCard = useFeature(feature.bookmarkOnCard);
```

Comment:
We have this currently in the post option menu dynamically?
Cant recall if we just always show it or based on experiment

Reply:
From the ticket Nensi created, she [mentioned](https://dailydotdev.slack.com/archives/C061T644XV3/p1700551627401439) it to be dynamically available.

Reply:
Yup, this should be dynamically available also in the share post options menu

Reply:
Cool. Let me apply it on that end too.